### PR TITLE
all departments page V1

### PR DESCRIPTION
**Pull request by Mohit-Sony**

### CODE REVIEW

File: configs/menuAside.js [hunk 1]
```diff
@@ -70,6 +70,19 @@ export default [
...
+  {
+    label: "Career",
+    icon: mdiFormDropdown,
+    menu: [
+      {
+        label: "All Departments",
+        to:'/careers/allDepartments'
+      },
+      {
+        label: "Dropdown item Dos",
```
Comment (panthsolanki) [inline configs/menuAside.js]: what is this label used for ?

File: components/Careers/SingleDepartment.vue
Comment (panthsolanki) [inline components/Careers/SingleDepartment.vue:1]: Rename file as "DepartmentCard" or something validate the content of the component

File: pages/careers/allDepartments.vue [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+</script>
+<style scoped>
+.container {
+  width: 100%;
+  margin-right: auto;
+  margin-left: auto;
+  padding-right: 12px;
+  padding-left: 12px;
+}
+@media (min-width: 540px) {
```
Comment (panthsolanki) [inline pages/careers/allDepartments.vue:141]: Please try to use tailwind for responsive css. for ref https://tailwindcss.com/docs/responsive-design

File: configs/menuAside.js (hunk 1 above)
Comment (Mohit-Sony) [reply configs/menuAside.js]: nothing its just demo for further things

Comment (panthsolanki) [reply configs/menuAside.js]: please remove it from this feature

File: pages/careers/allDepartments.vue (hunk 2 above)
Comment (Mohit-Sony) [reply pages/careers/allDepartments.vue:141]: this is for only one class only but if you still want then i can change

File: components/Careers/SingleDepartment.vue
Comment (Mohit-Sony) [reply components/Careers/SingleDepartment.vue:1]: ok

**Pull request merged**